ocrspace/main.py

```diff
@@ -28,25 +29,45 @@ class Language:
     Turkish = 'tur'
 
 
+@unique
```

Comment:
I don't think you need to use the unique decorator here

Reply:
resolved on c9ffd78